src/util/Timer.cpp

```diff
@@ -270,6 +285,7 @@ VirtualClock::crank(bool block)
     {
         std::lock_guard<std::recursive_mutex> lock(mDispatchingMutex);
         mDispatching = true;
+        mLastDispatchStart = std::chrono::steady_clock::now();
```

Comment:
Do we want to do this as well before `progressCount += mIOContext.poll_one();` and `progressCount += mActionScheduler->runOne();` as the goal is to give a fair quantum to each dispatch type.

Reply:
Fixed.